bigtable_input.py

```diff
@@ -599,6 +608,33 @@ def set_fresh_watermark(game_queue, count_from, window_size,
         game_queue.require_fresh_games(num_to_play)
 
 
+def mix_by_decile(games, moves, deciles=9):
+    """Compute a mix of regular and calibration games by decile.
+
+    deciles should be an integer between 0 and 10 inclusive.
+    """
+    assert 0 <= deciles <= 10
+    # The prefixes and suffixes below have the following meanings:
+    #   ct_: count
+    #   fr_: fraction
+    #    _r: resign (ordinary)
+    #   _nr: no-resign
+    lesser = 10 - math.floor(deciles)
+    greater = 10 - lesser
+    ct_r, ct_nr = greater, lesser
+    ct_total = ct_r + ct_nr
```

Comment:
I don't think ct_total is really needed, since it's always 10

Reply:
Good point. I'll rearrange it so that `ct_total` is defined up top as 10 and used like a local constant.